packages/remirror__extension-entity-reference/src/types.ts

```diff
@@ -24,6 +24,7 @@ export interface EntityReferenceOptions {
    */
   getStyle?: (entityReferences: EntityReferenceMetaData[][]) => Decoration[];
   blockSeparator?: AcceptUndefined<string>;
+  onClickMark?: (id: string) => void;
```

Comment:
Could you pls add a short comment how this works. My first reaction was: “Why do we declare a handler but never use it?”
I guess that’s because there is some underlying class that actually does something with it - and our options before simple.
What might be even better to bring the point across: extend `EntityReferenceOptions` from the option of this underlying class (just picking `onClickMark`). This would communicate very clearer to the reader of the code why the handler is there.